docs/src/main/paradox/ftp.md

```diff
@@ -86,6 +86,35 @@ This sink will consume @scaladoc[ByteString](akka.util.ByteString) elements and
 
 For both FTPs and SFTP servers, you will need to use the `FTPs` and `SFTP` API respectively.
 
+### Removing files
+
+In order to remove a remote file, you need to use the `remove` method in the FTP API:
+
+Scala
+: @@snip ($alpakka$/ftp/src/test/scala/akka/stream/alpakka/ftp/scalaExamples.scala) { #removing }
+
+Java
+: @@snip ($alpakka$/ftp/src/test/java/akka/stream/alpakka/ftp/examples/FtpRemovingExample.java) { #removing }
+
+This sink will consume @scaladoc[FtpFile](akka.stream.alpakka.ftp.FtpFile) elements and materializes to @scaladoc[Future](scala.concurrent.Future) in Scala API and @extref[CompletionStage](java-api:java/util/concurrent/CompletionStage) in Java API of @scaladoc[IOResult](akka.stream.IOResult) when the stream finishes.
```

Comment:
Please use `@javadoc` (same notation as `@scaladoc`).

Reply:
Thanks, I will update the PR further with the suggested changes and example in the weekend when I'm back from scala exchange

Reply:
@ennru 
I provided a Java and Scala example of downloading a file and then moving it. Is this what you had in mind? The case for which I am using the move sink is slightly different: I have a flow through which I send the bytes so in that case I could use via(..).

As for the @javadoc references, I'm not quite sure which references you mean. In other parts of ftp.md there are not @javadoc references.

Reply:
For referencing standard Java API docs, you may use `@javadoc` instead of `@extref`. We've set it up recently, that's why the rest of the FTP docs don't use it yet.

Reply:
@ennru Thanks for the clarification. I changed the ```@extrareff``` to ```@javadoc``` also.